{"uuid": "a375ae63-1ed7-4b7f-8ddc-367a33ac1c75", "children": ["3b07669e-75d7-4eff-9421-14a78c23ea94"], "befores": [{"name": "db_conn_params", "status": "broken", "statusDetails": {"message": "NameError: name 'DB_HOST' is not defined\n", "trace": "  File \"C:\\Python310\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Python310\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Python310\\lib\\site-packages\\_pytest\\fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\user\\Desktop\\projects\\MY_GITHIB\\qa_practise\\conftest.py\", line 42, in db_conn_params\n    return (DB_HOST, DB_NAME_DICT['INVENTORY'], DB_USER, DB_PASSWORD)\n"}, "start": 1686645209911, "stop": 1686645209911}], "start": 1686645209911, "stop": 1686645209929}